src/lib/keyboard.ts

```diff
@@ -102,7 +102,7 @@ export class KeyboardHandler {
   };
 
   bindAll = (): void => {
-    for (const key in this.keyMap) {
+    for (const [key] of Object.entries(this.keyMap)) {
```

Comment:
why? wait i dont get it

Reply:
Not strictly necessary but it's safer

Reply:
This revised version is wrong; fixing. You can file it as an issue

Reply:
The advantage of this was to be able to do
```js
for (const [key, value] of Object.entries(this.keyMap)) {
  this.bind(key, value);
}
```
but Object.entries doesn't infer types properly. Therefore, I can either do
```js
for (const [key, value] of Object.entries(this.keyMap)) {
  this.bind(key, value as Action);
}
```
or
```js
for (const key of Object.keys(this.keyMap)) {
  this.bind(key, this.keyMap[key]);
}
```
The first is faster